x-pack/plugins/security_solution/public/management/components/console/components/console_manager/components/console_page_overlay.tsx

```diff
@@ -6,21 +6,25 @@
  */
 
 import type { ReactNode, MouseEventHandler } from 'react';
-import React, { memo, useCallback, useMemo } from 'react';
+import React, { memo, useCallback, useMemo, useState } from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiButton, EuiButtonEmpty } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import type { PageLayoutProps } from './page_layout';
 import { PageLayout } from './page_layout';
 import { useTestIdGenerator } from '../../../../../hooks/use_test_id_generator';
 import { PageOverlay } from '../../../../page_overlay/page_overlay';
+import { ConsoleExitModal } from './console_exit_modal';
 
 const BACK_LABEL = i18n.translate('xpack.securitySolution.consolePageOverlay.backButtonLabel', {
   defaultMessage: 'Return to page content',
 });
 
 export interface ConsolePageOverlayProps {
+  agentId: string;
   console: ReactNode;
+  hasPendingActions: boolean;
+  hostName: string;
```

Comment:
So with the changes proposed above, the `agentId` and `hostName` will not be needed

Reply:
58fabe66865f7d1e624cac64bd153792ace085e8